vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/bitflyer/post_transaction/post_transaction_bitflyer.cc

```diff
@@ -64,6 +64,11 @@ type::Result PostTransaction::CheckStatusCode(const int status_code) {
     return type::Result::NOT_FOUND;
   }
 
+  if (status_code == net::HTTP_CONFLICT) {
```

Comment:
Could you make sure this status code is reflected in the header comment for this file too?

Reply:
Sure!